deeprank2/query.py

```diff
@@ -198,15 +199,10 @@ def _get_atom_node_key(atom) -> str:
     @property
     def model_id(self) -> str:
         """The ID of the model, usually a .PDB accession code."""
-        return self.model_id
+        return self._model_id
     @model_id.setter
-    def model_id(self, value):
-        self.model_id = value
-
-    @property
-    def targets(self) -> Dict[str, float]:
```

Comment:
we removed it because it is extremely clunky to use properties in dataclasses when they are input arguments. It can still be inspected, even if not a property. It's just less protected, but we agreed (after consulting with Cunliang) that this would be outweighed by the benefit of having more readable code using dataclasses without properties.